doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
+    namespace: App\Migrations
+    table_name: migration_versions
+    name: Application Migrations
```

Comment:
please remove the table_name and the name from this config, as requested in #57. We don't need to configure them explicitly in the project.

I would even remove the explicit configuration of the namespace here, keeping the default one. I don't see any reason to change it here (one could argue that you want to make the namespace fit your app, but your config will *not* make it compatible with PSR-4 anyway so this is a bad argument. And when using advanced feature to organize files per date instead of having a single folder with hundred of migrations, you *cannot* make code inside this folder fit PSR-4 at all, so I would not even try it).
This would then leave only the `dir_name` config (which is the one we really need due to the new Flex structure)